.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,34 @@
+name: Unit test for Pyodide
+on:
+  push:
+  pull_request:
+
+env:
+  FORCE_COLOR: 3
```

Comment:
Out of curiosity do you know what this does?

Reply:
FORCE_COLOR is a (proposed) standard environment variable akin to `NO_COLOR` that offers the entire spectrum of 24-bit colour support if set to `3`. CI providers like CircleCI offer coloured support for their logs by default, but GitHub Actions does not.

See https://force-color.org/ for more!